server/server.go

```diff
@@ -68,6 +80,7 @@ const (
 	AuthorizationOff AuthorizationScheme = iota
 	AuthorizationBasic
 )
+const defaultMinTLSVersion = tls.VersionTLS12
```

Comment:
Let's add a blank line above this. Right now, it seems like it had something to do with the authorization schemas.

Reply:
Done